src/client/article.ts

```diff
@@ -125,7 +126,18 @@ function slideshow(): void {
         }));
 }
 
+function formatDates(): void {
+    Array.from(document.querySelectorAll('time[data-date]'))
+        .forEach(time => {
+            const timestamp = time.getAttribute('data-date');
+            if (timestamp) {
+                time.textContent = formatDate(new Date(timestamp))
```

Comment:
you're quite likely to hit an exception when trying to parse a date from a string, I'd make sure this is handled gracefully, as one mis-formatted date would stop the whole page from rendering.